charts/exivity/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 
-version: "3.33.3"
+version: "3.33.2"
```

```suggestion
version: "3.33.3"
```

Do not merge until version 3.33.3 is set again